src/components/Popup/Popup.tsx

```diff
@@ -411,9 +418,11 @@ export default class Popup extends AutoControlledComponent<ReactProps<PopupProps
     popupPositionClasses: string,
     rtl: boolean,
     accessibility: AccessibilityBehavior,
+    // https://popper.js.org/popper-documentation.html#Popper.scheduleUpdate
     { ref, scheduleUpdate, style: popupPlacementStyles }: PopperChildrenProps,
   ) => {
-    const { content } = this.props
+    const { content: propsContent, renderContent } = this.props
+    const content = renderContent ? renderContent(scheduleUpdate) : propsContent
```

Comment:
Not sure but, maybe it will be nice if we can have an example with this, especially that we are adding some new prop for the API. Other then that looks good!

Reply:
totally agree, will work on it!